src/zenml/post_execution/step.py

```diff
@@ -134,6 +134,69 @@ def step_configuration(self) -> Dict[str, Any]:
         """
         return self._model.step_configuration
 
+    @property
+    def settings(self) -> Dict[str, Any]:
+        """Returns the step settings.
+
+        Returns:
+            The step settings.
+        """
+        settings = self.step_configuration["config"]["settings"]
+        return cast(Dict[str, Any], settings)
+
+    @property
+    def extra(self) -> Dict[str, Any]:
+        """Returns the extra dictionary.
+
+        Returns:
+            The extra dictionary.
+        """
+        extra = self.step_configuration["config"]["extra"]
+        return cast(Dict[str, Any], extra)
+
+    @property
+    def enable_cache(self) -> bool:
+        """Returns whether caching is enabled for this step.
+
+        Returns:
+            Whether caching is enabled for this step.
+        """
+        enable_cache = self.step_configuration["config"]["enable_cache"]
+        return cast(bool, enable_cache)
+
+    @property
+    def step_operator(self) -> Optional[str]:
+        """Returns the step operator of the step.
+
+        Returns:
+            The step operator of the step.
+        """
+        step_operator = self.step_configuration["config"]["step_operator"]
+        return cast(Optional[str], step_operator)
+
+    @property
+    def experiment_tracker(self) -> Optional[str]:
+        """Returns the experiment tracker of the step.
```

Comment:
Returns the `name of the experiment tracker`

Reply:
Good catch, done!